java-admanager/pom.xml

```diff
@@ -46,4 +46,4 @@
     <module>ad-manager-bom</module>
   </modules>
 
-</project>
+  </project>
```

Comment:
Can we revert this change as well? Also do we know where this change is from?

Reply:
This also comes from `fix-poms.py`.

pom.xml is not rendered from templates when updates so it's not clear to me how this indentation come.